practitioner-service/README.md

```diff
@@ -71,29 +82,118 @@ Creates a new practitioner entity.
 
 ```json
 {
-    "error": "Fhir store connection failed with error..."
+  "error": "Fhir store connection failed with error..."
 }
 ```
+
+### Create RoleAssignment
+
+Assigns a role to a practitioner in a site.
+
+**URL** : `/practitioner/{practitionerId}/roles`
+
+**Payload** :
+
+```json
+{
+  "siteId": "mySiteId",
+  "roleId": "myRoleId"
+}
+```
+
+| Parameter | Data Type Expected |                                                       Description                                                      |
+|:---------:|:------------------:|:----------------------------------------------------------------------------------------------------------------------:|
+|   practitionerId  |       string       | The id of the practitioner |
+|   siteId          |       string       | The id of the site the role will apply to |
+|   roleId          |       string       | The id of the role to assign |
+
+**Method** : `POST`
+
+#### Success Response
+
+**Condition** : If the role was assigned successfully in the FHIR store.
+
+**Code** : `201 CREATED`
+
+**Content example**
+
+```json
+{
+  "id": "787485bc-7c4e-4d2d-bf42-f9db010e5fb5"
```

Comment:
I think after I've changed the Response class to a generic class the response is not with "id" property but "object" property, if we want it to be id we should consider adding another fixed property for the Id.
Current response for Response<String> is:
```
{
  "object": "787485bc-7c4e-4d2d-bf42-f9db010e5fb5"
 }
```

Reply:
1. This readme version represents what's currently on the main branch.
2. Why "object"? seems very generic and doesn't say what it is...